android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java

```diff
@@ -1122,6 +1122,9 @@ public void apmNetworkLogByReflection(HashMap<String, Object> jsonObject) throws
         //--------------------------------------------
         final String requestContentType = (String) jsonObject.get("contentType");
         final String responseContentType = (String) jsonObject.get("responseContentType");
+        //--------------------------------------------
+         final long requestBodySize = ((Number) jsonObject.get("requestBodySize")).longValue();
+         final long responseBodySize = ((Number) jsonObject.get("responseBodySize")).longValue();
```

Comment:
@ahmed1hisham many parsing and casting operations are being done here. Could we please wrap the whole function in a try-catch block?

Reply:
Sure, on it